crates/rspack_binding_options/src/options/raw_split_chunks.rs

```diff
@@ -303,6 +314,43 @@ impl From<RawSplitChunksOptions> for new_split_chunks_plugin::PluginOptions {
   }
 }
 
+#[derive(Derivative, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[napi(object)]
+#[derivative(Debug)]
+pub struct RawSplitChunksSizesOptions {
+  pub javascript: Option<f64>,
+  pub css: Option<f64>,
+  pub wasm: Option<f64>,
+  pub asset: Option<f64>,
+  // 不知道webpack里是如何转换的？我先遵循之前的实现就行吧？
+  pub unknown: Option<f64>,
+}
+
+impl RawSplitChunksSizesOptions {
+  pub fn to_hash_map(&self) -> FxHashMap<SourceType, f64> {
```

Comment:
This is a good place to do a little exhaustiveness checking but I'm not familiar with rust, any recommendations on how to write it?